bioio/tests/test_plugins.py

```diff
@@ -16,7 +16,8 @@ def test_dump_plugins(dummy_plugin: str) -> None:
     old_stdout = sys.stdout
     sys.stdout = StringIO()
     try:
-        dump_plugins()
+        # Disable plugin caching to prevent other tests from interfering
+        dump_plugins(use_cache=False)
```

Comment:
This was necessary because the test I added doesn't use the dummy plugin and runs before this test.